packages/@adobe/spectrum-css-temp/components/inputgroup/index.css

```diff
@@ -44,7 +44,8 @@ governing permissions and limitations under the License.
     flex-shrink: 0;
   }
 
-  &.is-disabled {
+  /* Quiet or invalid inputgroup field button should always have a border width. */
+  &.is-disabled:not(.spectrum-InputGroup--invalid):not(.spectrum-InputGroup--quiet) {
```

Comment:
have you run chromatic on this branch? i recall doing something like this that broke something else, i don't recall what it was, but i think it came up in chromatic

Reply:
yep, you can take a look at https://www.chromatic.com/build?appId=5f0dd5ad2b5fc10022a2e320&number=77 (this is the run before the addition of `:not(.spectrum-InputGroup--quiet)`. At first I had `&.is-disabled:not(.spectrum-InputGroup--invalid)` only which broke quiet + read only comboboxes.

https://www.chromatic.com/build?appId=5f0dd5ad2b5fc10022a2e320&number=78 is the final run

Reply:
great, thanks! i approved the tooltip ones, at some point need to figure out why those are flaky, i'll open an issue